src/linter.js

```diff
@@ -89,13 +89,17 @@ function extractModifierDomains(modifier) {
  *
  * @param {agtree.NetworkRule} ast - The AST of a network rule to extract
  * domains from.
+ * @param {Array<string>} ignoreDomains - List of domains to ignore.
  * @returns {Array<string>} The list of all domains that are used by this rule.
  */
-function extractNetworkRuleDomains(ast) {
+function extractNetworkRuleDomains(ast, ignoreDomains) {
```

Comment:
We need to ignore domains from modifiers, e.g.
```adblock
/foo.js$domain=example.com
```

Also, please do not forget tests

Reply:
Hey,
thank you for that i forgot about it.
I did add some tests